ct/vikunja.sh

```diff
@@ -67,7 +67,7 @@ if [[ ! -f /opt/${APP}_version.txt ]] || [[ "${RELEASE}" != "$(cat /opt/${APP}_v
 
   msg_info "Updating ${APP} to ${RELEASE}"
   cd /opt
-  rm -rf /opt/*
+  rm -r /opt/vikunja/vikunja
```

Comment:
```suggestion
  rm -rf /opt/vikunja/vikunja
```
Add `f` to avoid crash if it file does not exist. Recursive `r` option is not really needed, but better to just have it on every rm, so if someone copies it for a folder we don't have to correct them

Reply:
You are right, is changed.